### DISCUSSION TITLE
How I can commit without using mouse?

### DISCUSSION THREAD

**Discussion by MairwunNx** [gitextensions/gitextensions#9928] [Q&A] [answered]
I want to less use mouse with Git Extensions, I trying to commit with `ctrl` + `space` but next, I want to type message and click on `commit` button without mouse, how I can do that? I tried to navigate to this with tabs but focus always stay in the text field.

**Comment by pmiossec** [comment, marked as answer]
>I want to less use mouse with Git Extensions, I trying to commit with `ctrl` + `space` but next, I want to type message and click on `commit` button without mouse, how I can do that?

The solution is definitively, inside the commit message, to do: <kbd>Ctrl</kbd>+<kbd>Enter</kbd>

In #3920 you could find also some improvements to use commit form with the keyboard but once you are in the commit message textarea it's difficult to go out because it is writing a tab character.
You could use still <kbd>Ctrl</kbd>+<kbd>Tab</kbd> to go out (of a textarea --shortcut hardly known--) but the "Commit" button has no tab stop so it will not help.

**Reply by RussKie** [reply]
> but once you are in the commit message textarea it's difficult to go out because it is writing a tab character.

The buttons have keyboard accelerators (i.e., <kbd>ALT+\<key\></kbd>):
![image](https://user-images.githubusercontent.com/4403806/161377602-d358545a-aa60-4a2e-9754-ac1f6a0adf31.png)


> the "Commit" button has no tab stop so it will not help.

This is a major usability problem, and it should be fixed. 
@MairwunNx please raise a new issue for this. A PR with a fix is also welcome.

**Comment by mstv** [comment]
GE has some more hotkeys. F.i. `Ctrl+4` focuses the commit message.

![grafik](https://user-images.githubusercontent.com/36601201/161400366-2d20f820-6471-4fa9-933a-cb1de50cf2dc.png)
